src/pages/dashboard.tsx

```diff
@@ -76,29 +84,29 @@ export function TableHero() {
   }, []);
 
   return (
-    <div className="p-4 md:p-10 rounded-2xl bg-gray-900">
-      <h1 className="text-gray-300 text-5xl font-semibold mb-2 mt-2 px-10">
+    <div className="p-4 md:p-10 rounded-2xl my-12">
+      <h1 className="text-white hover:text-secondary text-4xl md:text-5xl font-bold md:mb-4 capitalize">
```

Comment:
same here with the text color, set it up at the top if it's necessary, but I believe the sections also default text to dark (white)